libs/execution_context/ebpf_program.c

```diff
@@ -2569,6 +2588,11 @@ ebpf_program_execute_test_run(
     test_run_context->completion_context = completion_context;
     test_run_context->completion_callback = callback;
 
+    // Limit maximum test runs to 1024 while fuzzing to prevent timeouts.
+#if defined(FUZZER_BUILD)
+    test_run_context->options->repeat_count = min(test_run_context->options->repeat_count, 1024);
+#endif
```

Comment:
To me this seems the wrong place to put test-specific ifdefs.
Why can't this be in the fuzzer code itself instead of in the execution context?
Or alternatively, why is it fuzzer specific?

Reply:
Fixed.